.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: objective-c
+
+# macOS and Xcode Version
+# Xcode 9.2 running on macOS 10.12
+osx_image: xcode9.2
+
+# Build dependencies
+install:
+  - brew install python3
+  - sudo gem install asciidoctor
+  - ( cd External && ./makeAll )
+
+# Cache built deps
+cache:
+  directories:
+    - External
+
+script:
+  - xcodebuild -scheme "MoltenVK (Debug)"
```

Comment:
@ePirat 

Can you also add the Travis badge to the README as @karl-lunarg recommends, and tracked in issue #75, please? Thanks.

Reply:
Sure!